website/docs/getting-started/setup/installation-guides/kubernetes/migrate-k8s.md

```diff
@@ -251,32 +245,37 @@ Goal: Change configurations in `values.yaml` after migrating from step 2, this w
       redis-statefulset-0                         1/1     Running   0          55m
       ```
 
-### 5. Import database
+### Import database
+Follow the below steps to import data from archive file into new Helm chart:
 
-Goal: Importing data from archive file into new Helm chart
+  1. Create `restore` directory in new pod
 
-* Steps
-  * Create `restore` directory in new pod.
-  * Copy archive file from local to new pod.
-    * Run `import_db` command.
-* Command
-  *   Create `restore` directory in new pod
-
-      ```
+      ```bash
       kubectl exec appsmith-0 -- mkdir -p /appsmith-stacks/data/restore
       ```
-  *   Copy archive file from local to new pod:
+  2. Copy archive file from local to new pod:
 
-      ```
+      ```bash
       kubectl cp appsmith-data.archive appsmith-0:/appsmith-stacks/data/restore
       ```
 
-      * Run `import_db` command:
+ 3. Run `import_db` command:
 
-      ```
+    ```bash
       kubectl exec -it appsmith-0 -- appsmithctl import_db
-      ```
+    ```
+
+ You see a message - `Importing this DB will erase this data. Are you sure you want to proceed`, where you can respond with `y`. It's safe in this situation since the new database in the new setup only contains initial data and should be safe to be overwritten.
+
+4. Verify that Helm chart still works after importing and the data from old Kubernetes stack also shows up in the Helm chart
+
+## Migrate SSL certificate
+You may want to migrate existing SSL certificate to the new Helm chart. With the old Kubernetes stack & new Helm chart, Kubernetes cluster uses [`cert-manager`](https://cert-manager.io/) to provision the SSL certificate. `cert-manager` is an `Automate certificate manager` that provisions and manages the certificates itself => Backward incompatible if migrating certificate from one `cert-manager` to another one.
+
+It's recommended that after migrating to the Helm chart, you can follow the document [Setup Https](https://github.com/appsmithorg/appsmith/blob/release/deploy/helm/Setup-https.md) to setup a new `cert-manager` and provision a new certificate for Helm installation.
+
+## Test Appsmith
+After the chart has been deployed to the Kubernetes cluster, it's important to test the Appsmith application to ensure it's functioning as expected in the new Helm-based deployment. This may involve running a series of integration tests, or simply verifying that the application is accessible and responding to requests as expected.
```

Comment:
```suggestion
After the chart has been deployed to the Kubernetes cluster, it's important to test that the migration is successful by checking that the Application application is functioning as expected in the new Helm-based deployment. This may involve running a series of integration tests, or simply verifying that the application is accessible and responding to requests as expected.
```